openvdb_points/unittest/TestPointConversion.cc

```diff
@@ -551,6 +553,163 @@ TestPointConversion::testStride()
 }
 
 
+////////////////////////////////////////
+
+
+void
+TestPointConversion::testAutoVoxelSize()
```

Comment:
Looks good. I would suggest adding a few more examples to this: 

* one with points in a very small volume, for example all in 1e-7x1e-7x1e-7 to test the limits of tolerance as mentioned elsewhere
* one with a sparse data set, for example 1000 points in a 1x1x1 cube at the origin, 1000 points in a 1x1x1 cube at y=100, etc
* one with a non-uniform data set, for example 1000 points scattered in a 1x1x1 cube with 1000 points scattered in a 10x10x10x cube with 1000 points scattered in a 100x100x100 cube, etc